great_expectations/rule_based_profiler/parameter_builder/simple_date_format_string_parameter_builder.py

```diff
@@ -98,7 +98,7 @@ def __init__(
         threshold: Union[str, float] = 1.0,
         candidate_strings: Optional[Union[Iterable[str], str]] = None,
         evaluation_parameter_builder_configs: Optional[List[dict]] = None,
-        json_serialize: bool = True,
+        json_serialize: Union[str, bool] = True,
```

Comment:
Hi @alexsherstinsky `json_serialize` is a little confusing to me.  Could you tell me more about how it's intended to be used? In the tests I currently see `True`, `False`, and `None`

Reply:
@Shinnnyshinshin It controls the output format of ParameterBuilder -- expectations want serialized to JSON; internal dependencies must get numerically computable (hence, raw/non-serialized).  Thanks.